lago_python_client/models/usage_threshold.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Dict, List, Optional
+
+from lago_python_client.base_model import BaseModel
+
+from ..base_model import BaseResponseModel
+
+
+
+class UsageThreshold(BaseModel):
+    id: Optional[str]
+    threshold_display_name: Optional[str]
+    amount_cents: Optional[int]
+    recurring: Optional[bool]
+
+class UsageThresholds(BaseModel):
```

Comment:
Could you add one more empty line between classes? (I'm not a python expert but it looks like it is a common practice)

Reply:
I'll check if this can be added to a .prettierrc 👍